irohad/main/impl/pg_connection_init.cpp

```diff
@@ -151,7 +151,18 @@ PgConnectionInit::prepareWorkingDatabase(
     StartupWsvDataPolicy startup_wsv_data_policy,
     const PostgresOptions &options) {
   return getMaintenanceSession(options) | [&](auto maintenance_sql) {
-    if (startup_wsv_data_policy == StartupWsvDataPolicy::kReuse) {
+    int work_db_exists;
```

Comment:
It should be bool but soci does not accept bool. int works as logical boolean here